cds/records.bal

```diff
@@ -224,14 +224,27 @@ public type Card record {|
     }
     string summary;
     string detail?;
-    string indicator;
+    Indicator indicator;
     Source 'source;
     Suggestion[] suggestions?;
-    string selectionBehavior?;
+    SelectionBehavior selectionBehavior?;
     r4:Coding[] overrideReason?;
     Link[] links?;
 |};
 
+# Urgency/importance of what this card conveys.
+public enum Indicator{
```

Comment:
Are these values coming from the spec?

Reply:
Yes.. https://cds-hooks.hl7.org/2.0/#card-attributes:~:text=info%2C%20warning%2C%20critical